auto-ipsec/racoon/src/local_action_update_crl.py

```diff
@@ -43,7 +43,7 @@ async def execute(json_revocation):
         oldcrl = f.read()
 
     updated = False
-    for i in range(10):
+    for _ in range(10):
```

Comment:
I kind of feel this type of lint is overchecking, rarely seen tbh

Reply:
`i` is unused ...

Reply:
yeah, it's not used, just never hit this kind of comlains :)